Documentation/git-apply.txt

```diff
@@ -66,7 +66,7 @@ OPTIONS
 	disables it is in effect), make sure the patch is
 	applicable to what the current index file records.  If
 	the file to be patched in the working tree is not
-	up to date, it is flagged as an error.  This flag also
+	up-to-date, it is flagged as an error.  This flag also
```

Comment:
Are we ready to drop this "up-to-date" change? It had an interaction with VS. @dscho @jrbriggs ?

Reply:
We updated both VS 15.0 and 15.7+ with the fix to accept both "Already up-to-date" or "Already up to date" from the output of `git-merge` and `git-pull`.

Reply:
I opened a PR to drop this Revert: https://github.com/Microsoft/git/pull/72